node_filesystem/templates/bird.cfg.template

```diff
@@ -46,11 +46,23 @@ protocol direct {
   interface -"cali*", "*"; # Exclude cali* but include everything else.
 }
 
+{{/* ------------ Template definitions ----------------- */}}
+{{define "MESH4"}}
+# Peering with all neighbours (mesh).
+{{range $host := lsdir "/host"}}{{$onode_as_key := printf "/host/%s/bgp_as" .}}{{$onode_ip_key := printf "/host/%s/bird_ip" .}}{{$onode_ip := getv $onode_ip_key}}{{$nums := split $onode_ip "."}}{{$id := join $nums "_"}}
```

Comment:
No, not really - just results in more blank lines in the templates, but since we already have that a few more won't hurt.